### PR TITLE
'removed unused libraries'

### PR DESCRIPTION
**Pull request by matthewblackbu**
## Description
The first PR changes the Scraper documentation to reflect what we changed in the get_panel function
The second change removes all unused libraries and imports in different files across the ds repo 

Fixes # (issue)
The first PR will hopefully help the new cohort understand the changes we made to the get_panel function better since it isn't intuitive by the name anymore
The second PR will helpfully get us further from the storage cap and give us a little more wiggle room when it comes to getting more libraries later

first PR 
https://www.loom.com/share/535e433e4262423691eef7ed69495ad0

second PR
https://www.loom.com/share/6258957a87714a60a8170a330165068b
(had to cut I lost my train of thought)

## Type of change
- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] This change requires a documentation update
- [x] This is a documentation change and an efficiency change
## Checklist:
- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have removed unnecessary comments/console logs from my code
- [x] I have made corresponding changes to the documentation if necessary (optional)
- [x] My changes generate no new warnings
- [x] I have checked my code and corrected any misspellings
- [x] No duplicate code left within changed files
- [x] Size of pull request kept to a minimum
- [x] Pull request description clearly describes changes made & motivations for said changes
HERE ARE THE CHANGES NOT BEING USED:
beautifulsoup
numpy
visualizations
plotly express
json

### CODE REVIEW

**Comment by matthewblackbu** [comment]
I tried to make another branch for the second commit and It just changed the name of the first one to that one and added the commit to it Sorry everyone if this is confusing If someone could DM me with a better way to go about multiple different PRs that would be great!

**Comment by matthewblackbu** [comment]
Hopefully this is the correct way to go about this I made a branch of a branch and the second one took the PRs from the first but what's most important is the second PR here

**Review by ctscofield — APPROVED** [review]
Nice cleanup getting rid of the unused libraries

**Review by AZICO — APPROVED** [review]
Helpful information for the next cohort.

File: SCRAPER_INFO.md [hunk 1]
```diff
@@ -34,18 +34,11 @@
...
-- **Current Status:** Merged and current.
-
-### Judges
-- **Function name: **`get_judges()`
 - **Details:** Returns judge presiding over the original decision- this is located at the bottom of an original decision, if one exists in the document. Judges are not mentioned in appeals cases so those cases will return 'Null'
 - **Returns:** String like ‘Clarease Rankin Yates’
-- **Current Status:** There has been ongoing discussion about how to best pull out this field and the get_pannel field. When we inherited this code, every time the function was run it was scraping data about the judges from a Wikipedia page. As of 4/27, the conclusion was that we needed to create a database or some sort of storage in the backend to hold a static list of these judges. This has not been started yet. A basic implementation can be found in [this notebook](https://github.com/Lambda-School-Labs/human-rights-first-asylum-ds-a/blob/main/notebooks/get_judge_implementation.ipynb).
-
+- **Current Status:** WIP returning the first member in the panel as the judge This should be the primary judge making the deciusion.
```
Comment (jihoover77) [inline SCRAPER_INFO.md]: There is a spelling error deciusion should be decision.

File: SCRAPER_INFO.md [hunk 2]
```diff
@@ -38,7 +38,8 @@
 - **Function name: **`get_panel()`
 - **Details:** Returns judge presiding over the original decision- this is located at the bottom of an original decision, if one exists in the document. Judges are not mentioned in appeals cases so those cases will return 'Null'
 - **Returns:** String like ‘Clarease Rankin Yates’
-- **Current Status:** WIP returning the first member in the panel as the judge This should be the primary judge making the deciusion.
+- **Current Status:** Work In Progress returning the first member in the panel as the judge This should be the primary judge making the decision.
```
Comment (jihoover77) [inline SCRAPER_INFO.md:41]: Great job you fixed the spelling error

**Review by jihoover77 — COMMENTED** [review]
Nice clean up of the code base.

**Review by Vehmeyer — APPROVED** [review]
Nice work. It's always good to remove unused libraries in order to keep the code as clean as possible. I appreciate that you included screen shots in the PR for quick reference.

**Pull request merged**